src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -537,8 +537,8 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
 
         if (decimals <= 18) {
             // If issuance token has 18 or fewer decimals. Ensure that decimal difference is at least 1
-            // to end up with minRequiredSupply of 10
-            decimalDifference = 18 - decimals == 0 ? 1 : 18 - decimals;
+            // i.e 10**0
+            decimalDifference = 0;
```

Comment:
`decimalDifference` is already zero by initialization, so this could be cheaper and simplified to
```Solidity
uint decimalDifference;
uint decimals = issuanceTokenDecimals;

// If the issuance token has 18 or fewer decimals, we leave the decimalDifference at 0.,
// such that 10**0 equals 1.
// If the issuance token has more than 18 decimals, we calculate the difference.
if (decimals > 18) {
    decimalDifference = decimals - 18;
}
```

Reply:
Updated it. Also renamed the variable for clarity @marvinkruse